packages/driver/src/cy/keyboard.ts

```diff
@@ -12,6 +12,7 @@ import type { HTMLTextLikeElement } from '../dom/elements'
 import $selection from '../dom/selection'
 import $utils from '../cypress/utils'
 import $window from '../dom/window'
+import type { Log } from '../cypress/log'
```

Comment:
These should pull from the existing Log Types

```suggestion
- import type { Log } from '../cypress/log'
```

Reply:
It's not possible because the `Log` class under `driver/src/cypress/log.ts` has some internal methods.

Here are some examples:

https://github.com/cypress-io/cypress/blob/68173be4aa3a9d6c64cc447aa68d8f374bc3c066/packages/driver/src/cy/commands/actions/click.ts#L138-L140

https://github.com/cypress-io/cypress/blob/68173be4aa3a9d6c64cc447aa68d8f374bc3c066/packages/driver/src/cy/assertions.ts#L465-L468